src/main/java/io/fabric8/maven/docker/util/AuthConfigFactory.java

```diff
@@ -439,6 +442,11 @@ private Reader getFileReaderFromHomeDir(String path) {
         } else {
             return null;
         }
+    }    
+    
+    private Reader getFileReaderFromHomeDir(String path) {
+        File file = new File(getHomeDir(),path);
+        return getFileReaderFromDir(file)
```

Comment:
missing semi-colon, but I suggest to inline this method anyway (using directly `getFileReaderFromDir(new File(getHomeDir(),path)` where used.